chat.js

```diff
@@ -96,6 +103,8 @@ async function submitRequest() {
           }
           responseDiv.hidden_text += word;
           responseDiv.innerHTML = DOMPurify.sanitize(marked.parse(responseDiv.hidden_text)); // Append word to response container
+          // autoscrolls chat container
+          scrollToBottom(chatContainer);
```

Comment:
Do we need this extra call to scroll to the bottom?
If the user has already scrolled up then they are probably looking at previous chats